chia/server/server.py

```diff
@@ -419,14 +419,8 @@ async def start_client(
             timeout_value = float(self.config.get("peer_connect_timeout", 30))
             timeout = ClientTimeout(total=timeout_value)
             session = ClientSession(timeout=timeout)
-
-            try:
-                if type(ip_address(target_node.host)) is IPv6Address:
-                    target_node = PeerInfo(f"[{target_node.host}]", target_node.port)
-            except ValueError:
-                pass
-
-            url = f"wss://{target_node.host}:{target_node.port}/ws"
+            ip = f"[{target_node.ip}]" if target_node.ip.is_v6 else f"{target_node.ip}"
+            url = f"wss://{ip}:{target_node.port}/ws"
```

Comment:
maybe in the future it would make sense to make `PeerInfo` have a `__str__()` that contains this to-string logic